tests/components/minecraft_server/test_config_flow.py

```diff
@@ -163,19 +222,17 @@
                 )
 
                 assert result["type"] == RESULT_TYPE_CREATE_ENTRY
-                assert (
-                    result["title"]
-                    == f"{USER_INPUT_IPV4[CONF_HOST]}:{USER_INPUT_IPV4[CONF_PORT]}"
-                )
+                assert result["title"] == USER_INPUT_IPV4[CONF_HOST]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5232399)